cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/DcLineSegmentConversion.java

```diff
@@ -30,32 +34,146 @@ public DcLineSegmentConversion(PropertyBag l, Context context) {
         cconverter2 = context.dc().cgmesConverterFor(iconverter2);
     }
 
+    private static double getPAc(PropertyBag p) {
+        return Double.isNaN(p.asDouble(TARGET_PPCC)) ? 0 : p.asDouble(TARGET_PPCC); // targetPpcc is the real power injection target in the AC grid in CGMES
+    }
+
+    private void updateLossFactor1(double pAC1, double poleLossP1, HvdcLine.ConvertersMode mode) {
+        if (mode.equals(HvdcLine.ConvertersMode.SIDE_1_RECTIFIER_SIDE_2_INVERTER)) {
+            iconverter1.setLossFactor((float) (poleLossP1 / pAC1) * 100);
+        } else if (pAC1 + poleLossP1 != 0) {
```

Comment:
I think we have to add in the else-if condition the fact that ````mode.equals(HvdcLine.ConvertersMode.SIDE_1_INVERTER_SIDE_2_RECTIFIER)````
In this case, pAC1 is negative (inverter side), so I think that Math.abs(pAC1) is required.

Reply:
pAC1's sign is **independent** of the fact of this HVDC converter station is inverter or rectifier, it is only dependent of the **side**

Reply:
Okay, can we add a comment to explain the CGMES convention (depending on the side and not the mode of the conversion station) ?

Reply:
It is already commented lines 184-185:
```
        // load sign convention is used i.e. positive sign means flow out from a node
        // i.e. pAC1 >= 0 and pAC2 <= 0
```

Reply:
About the comment in lines 184-185: I agree on the fact that load convention is used for defining `targetPpc`, but this not implies that `pAC1 >= 0 and pAC2 <= 0`. 

In the module we are assuming (method `decodeMode`) that the user either defines explicitly the mode of operation of each converter or we try to identify which one is converter and which one is rectifier depending on the sign of `targetPpcc` at both ends.

A converter with `targetPpcc > 0` is considered a rectifier (power going out from AC network is injected into the DC line). But the `targetPpcc > 0` could be at side 1 or side 2 of the DC line.

Maybe we could add an additional check to verify that, when the mode is defined explicitly the `targetPpcc` has the expected sign. That is, if the `operatingMode` is `rectifier`, then we would expect `targetPpcc > 0`.